### ISSUE TITLE
pandas fillna not returning correct type

### ISSUE THREAD

**Issue by Dr-Irv** [microsoft/pylance-release#457]
## Environment data

-   Language Server version: 2020.9.8
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): Anaconda Python 3.7.5

## Expected behaviour

No error

## Actual behaviour

```python
import pandas as pd
import numpy as np

df: pd.DataFrame = pd.DataFrame({"x": [1, np.nan, 3]})

ndf: pd.DataFrame = df.fillna(-1)

print(ndf)
```
reports
```
Expression of type "DataFrame | None" cannot be assigned to declared type "DataFrame"
  Type "DataFrame | None" cannot be assigned to type "DataFrame"
    Type "None" cannot be assigned to type "DataFrame"
```

Possible fix:
```
diff frame.pyi.ORIG frame.pyi
756c756
<         inplace: Literal[False]
---
>         inplace: Literal[False] = ...
```

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.10.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020100-7-october-2020

**Comment by Dr-Irv** [comment]
@jakebailey I am running the 20.10.0 version now, and this is *not* fixed. 

Here is what I did to that version to fix it:
```
diff frame.pyi.ORIG frame.pyi
653c653
<         inplace: Literal[True] = ...
---
>         inplace: Literal[True]
760c760
<         inplace: Literal[False]
---
>         inplace: Literal[False] = ...
```

**Comment by jakebailey** [comment]
Thanks, I'll fix that.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.10.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020101-14-october-2020
